### PR TITLE
[5.1] Request Accepts

### PR DESCRIPTION
**Pull request by jbrooksuk**
Thanks to @GrahamCampbell for implementation details here :+1:

### CODE REVIEW

File: tests/Http/HttpRequestTest.php [hunk 1]
```diff
@@ -432,19 +432,51 @@ public function testFlushMethodCallsSession()
...
+		$this->assertTrue($request->accepts('text/html'));
+		$this->assertTrue($request->acceptsHtml());
+		$this->assertFalse($request->acceptsJson());
+	}
+
+
+	public function testFormatReturnsAcceptsMultiple()
+	{
+		$request = Request::create('/', 'GET', [], [], [], ['HTTP_ACCEPT' => 'application/json,text/html']);
+		$this->assertTrue($request->accepts(['*/*', 'text/html', 'application/json']));
```
Comment (GrahamCampbell) [inline tests/Http/HttpRequestTest.php]: Maybe try this individually too?

Comment (GrahamCampbell) [inline tests/Http/HttpRequestTest.php]: Because under the hood we're already bailed out after the very first iteration.

Comment (jbrooksuk) [inline tests/Http/HttpRequestTest.php]: Ok.

Comment (jbrooksuk) [inline tests/Http/HttpRequestTest.php]: Done.

**Comment by joecohens** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
Unfortunately, this is broken. Will fix and re-PR. :)

**Comment by jbrooksuk** [comment]
Oh :(

**Pull request closed without merging**